content/blog/2018-05-02-onboarding-is-work.md

```diff
@@ -0,0 +1,325 @@
+             col = "grey50",
+             size = 0.2) +
+    scale_y_continuous(breaks= seq(min(-custom_transf(example$total_deletions)),
+                                   to = max(custom_transf(example$total_insertions)),
+                                   by = 5), 
+                       labels= round(custom_exp(seq(min(-custom_transf(example$total_deletions)),
+                                         to = max(custom_transf(example$total_insertions)),
+                                         by = 5))))+
+    annotate("rect",
+             xmin = example$opened[1], xmax = example$closed[1],
+             ymin = - custom_transf(max(example$total_deletions)), 
+             ymax = custom_transf(max(example$total_insertions)),
+             fill = "grey80", alpha = 0.6,
+             col = "grey10",
+             size = 0.2) +
+    ggtitle(package_name) +
+    hrbrthemes::theme_ipsum() 
+  print(p)
+}
+```
+
+Let’s have a look at a few packages.
+
+``` r
+plot_commits("opencage", commits = commits)
+```
+
+![commits plot of the opencage
+package](/img/blog-images/2018-05-02-onboarding-is-work/unnamed-chunk-2-1.png)
+
+``` r
+plot_commits("charlatan", commits = commits)
+```
+
+![commits plot of the charlatan
+package](/img/blog-images/2018-05-02-onboarding-is-work/unnamed-chunk-3-1.png)
+
+When looking at all onboarded repos, we don’t see a general pattern to
+commit histories. Commits have diffent sizes, and the activity frequency
+is highly variable. While we see the updates to packages that occur
+during review, there is just as often change again after onboarding, as
+packages continue to improve and be maintained. This highlights an
+important aspect of our peer review, which is that acceptance is not the
+end of the process. While we aim for the process to ensure high quality
+at acceptance, packages are not static moving forwards. Not all
+deficiencies are detected by peer reviewers, and further improvements
+and maintanance will continue. This is why many of our standards are
+about maintainability, and our process is in part an approach to onboard
+authors into a community of dedicated and supportive package
+maintainers.
+
+We do try to onboard mature packages, i.e. that are not *drafts*.
+Furthermore, the absence of general patterns in the previous figures
+could be due to different age at submission. How old are packages at
+submission?
+
+``` r
+age <- commits %>%
+  dplyr::group_by(package) %>%
+  dplyr::summarise(age = difftime(min(opened), min(datetime), units = "weeks")) %>%
+  dplyr::filter(age > 0)
+
+library(hrbrthemes)
+ggplot(age) +
+  geom_dotplot(aes(age), binwidth = 5) +
+  xlab("Age (weeks)") +
+  scale_y_continuous(NULL, breaks = NULL) +
+  hrbrthemes::theme_ipsum(base_size = 16,
+                          axis_title_size = 16)
+```
+
+![age of packages at
+submission](/img/blog-images/2018-05-02-onboarding-is-work/unnamed-chunk-4-1.png)
+
+This shows *apparent* age because the first commit might happen a long
```

Comment:
I would say "age at submission" here because there's a lot of space (code and plot) between saying you're going to look at age at submission and drawing the conclusion. Again, a bit of narrative explaining the plot would help a person who is not reading thoroughly but wants to get the take home messages